recipes/barrnap/0.3/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-mkdir -p $PREFIX
-mv ./* $PREFIX/
+mkdir -p $PREFIX/bin
+mv ./* $PREFIX/bin
```

Comment:
@johanneskoester again this is a not so ideal example for good installation guidelines.

From the readme:
```
## Install

    % cd $HOME
    % tar zxvf barnapp-XXX.tar.gz
    % echo 'PATH=$PATH:$HOME/barnapp-XXX' >> .bashrc
    (logout and log back in)
```

I can try to guess the correct folders and subfolders and not copy the readme if it's worth it. For example the example folders are considered useful?